app/lib/efile/ny/it215.rb

```diff
@@ -62,7 +62,7 @@ def calculate_line_16
 
       def calculate_wk_c_line_2
         rates = [
-          [0, 5000, nil, nil, 0.30],
+          [-99999999999999, 5000, nil, nil, 0.30],
```

Comment:
Question: What did this line do and why was it deleted?

Reply:
I think that Mike discovered that the deduction math can cause the agi to go negative, so this change means a rate can still be calculated in that scenario.